gradle-server/src/main/java/com/github/badsyntax/gradle/handlers/GetBuildHandler.java

```diff
@@ -252,4 +322,25 @@ private void replyWithStandardError(byte[] bytes) {
                     .setOutputBytes(byteString))
             .build());
   }
+
+  private void replyWithCompatibilityCheckError(
+      String gradleVersion, String javaVersion, int jdkMajorVersion) {
+    String errorMessage =
+        "Could not use Gradle version "
+            + gradleVersion
+            + " and Java version "
+            + javaVersion
+            + " to configure the build. Please consider either to change your Java Runtime to no more than Java "
```

Comment:
more -> higher. 

The error message is contructed by you, and I'm afraid users cannot easily find solutions by googling the message. One point is, can we also provide below link when this error occur?  https://docs.gradle.org/current/userguide/compatibility.html

Reply:
> more -> higher.

👍 
> One point is, can we also provide below link when this error occur? https://docs.gradle.org/current/userguide/compatibility.html

Sure. Can we offer an option named like "Learn More" in the error notification to link to the page?

Reply:
Do you know error type? If true, I think it's feasible to provide corresponding learn more button

Reply:
we can ensure it's a compatibility error or not. See https://github.com/microsoft/vscode-gradle/pull/931/files#diff-377d27baac4633ac0b67be3f25a54200bc498a997145152821d7d100da1b4ccfR212